### PR TITLE
Added notes for release 4.8.31

### PR DESCRIPTION
**Pull request by ctauchen**
These are the notes for release 4.8.31.

Applies only to `enterprise-4.8`

Preview:

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -3037,3 +3037,54 @@ link:https://access.redhat.com/solutions/6681391[{product-title} 4.8.29 containe
...
+
+link:https://access.redhat.com/solutions/6730091[{product-title} 4.8.31 container image list]
+
+[id="ocp-4-8-31-features"]
+==== Features
+
+[id="ocp-4-8-31-features-ip=reconciler"]
+===== IP reconciliation for Whereabouts CNI plug-in
+
+A new enhancement to the Whereabouts CNI plugin adds an IP reconciliation job, `ip-reconciler`, which runs as a Kubernetes cronjob.
```
Comment (adellape) [inline release_notes/ocp-4-8-release-notes.adoc:3058]: "plug-in" per IBMSG

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -3037,3 +3037,54 @@ link:https://access.redhat.com/solutions/6681391[{product-title} 4.8.29 containe
...
+{product-title} release 4.8.31 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:0484[RHBA-2022:0484] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:0483[RHSA-2022:0483] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6730091[{product-title} 4.8.31 container image list]
+
+[id="ocp-4-8-31-features"]
+==== Features
+
+[id="ocp-4-8-31-features-ip=reconciler"]
```
Comment (adellape) [inline release_notes/ocp-4-8-release-notes.adoc:3055]: `-` instead of `=` here?

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (ctauchen) [reply release_notes/ocp-4-8-release-notes.adoc]: Fixed! Thanks!

File: release_notes/ocp-4-8-release-notes.adoc (hunk 2 above)
Comment (ctauchen) [reply release_notes/ocp-4-8-release-notes.adoc]: Yes, thanks for that!

File: release_notes/ocp-4-8-release-notes.adoc [hunk 3]
```diff
@@ -3037,3 +3037,54 @@ link:https://access.redhat.com/solutions/6681391[{product-title} 4.8.29 containe
...
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-8-31"]
+=== RHBA-2022:0484 - {product-title} 4.8.31 bug fix and security update
+
+Issued: 2022-02-15
+
+{product-title} release 4.8.31 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:0484[RHBA-2022:0484] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:0483[RHSA-2022:0483] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
```
Comment (lpettyjo) [inline release_notes/ocp-4-8-release-notes.adoc:3048]: For the second sentence, it is generally advisable to put the prefacing phrase first, and then the xref: 

"For notes on the container images in this release, see the following article:"

File: release_notes/ocp-4-8-release-notes.adoc [hunk 4]
```diff
@@ -3037,3 +3037,54 @@ link:https://access.redhat.com/solutions/6681391[{product-title} 4.8.29 containe
...
+
+* Previously, the web console did not read the `resource` property for `ServiceBinding`-type resources.
+As a result, the topology view did not display service binding connectors.
+This update fixes the problem. The web console reads the `resource` property and correctly displays service binding connectors.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2019301[*BZ#2019301*])
+
+[id="ocp-4-8-31-upgrading"]
+==== Upgrading
+
+To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
```
Comment (lpettyjo) [inline release_notes/ocp-4-8-release-notes.adoc:3090]: I believe that the preferred term is "update", rather than "upgrade".

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

**Pull request merged**